lib/security_response_header_filter.js

```diff
@@ -59,7 +59,8 @@ module.exports = function addCSPHeaders(hyper, req, next, options) {
         // Set up basic CORS headers
         rh['access-control-allow-origin'] = '*';
         rh['access-control-allow-methods'] = 'GET';
-        rh['access-control-allow-headers'] = 'accept, content-type';
+        rh['access-control-allow-headers'] =
+            'accept, accept-encoding, origin, accept-language, content-type';
```

Comment:
It's actually impossible to override `origin` in clients, as it is part of verifying the domain comes from as part of [the CORS system](https://developer.mozilla.org/en-US/docs/Web/HTTP/Access_control_CORS). Adding it here would just be confusing.